packages/components/src/Card/Card.tsx

```diff
@@ -57,15 +57,7 @@ export function Card({
     <>
       {title && (
         <div className={styles.header}>
-          <Typography
-            element="h3"
-            size="large"
-            textCase="uppercase"
-            fontWeight="extraBold"
-            textColor="heading"
-          >
-            {title}
-          </Typography>
+          <Heading level={3}>{title}</Heading>
```

Comment:
This made me realize that `Modal` will need the same styling also, since it doesn't use `Card`.  We can update it here or I can create a new ticket, if you want to keep it all granular...

Reply:
I can do it here!